### ISSUE TITLE
CI failure on R-devel

### ISSUE THREAD

**Issue by yutannihilation** [extendr/libR-sys#149]
```
    thread 'main' panicked at 'Unknown kind: UnionDecl', build.rs:451:19
```
https://github.com/extendr/libR-sys/actions/runs/4584396694/jobs/8095855354#step:13:225

**Comment by yutannihilation** [comment]
It seems `Rcomplex` will be a union, not a struct in R 4.3.

https://github.com/r-devel/r-svn/commit/862f9f816ff3ff3cb3f851603f19e99f60a98475

It's just we didn't see any union in the source code, so the fix would be simple. However, I'm not sure if this affects extendr's complex-related features. Will check later.

**Comment by yutannihilation** [comment]
Hmm, it looks worrying...

https://github.com/search?q=repo%3Aextendr%2Fextendr%20rcomplex&type=code

**Comment by yutannihilation** [comment]
Closing in favor of https://github.com/extendr/extendr/issues/524, because the problem on libR-sys's side is now fixed by #150
